docs/install.md

```diff
@@ -0,0 +1,72 @@
+# Installing a node
+
+Right now 4 cores and 8 GB of RAM should be enough to run a node, but it might increase in the future. More info in the [FAQ](docs/faq.md).
+
+## From binaries
+If you just wish to run a Massa node without compiling it yourself, you
+can simply run the latest binary (download and execute):
+
+-   [Windows
+    executable](https://gitlab.com/massalabs/massa/-/jobs/artifacts/testnet/download?job=build-windows)
+-   [Linux
+    binary](https://gitlab.com/massalabs/massa/-/jobs/artifacts/testnet/download?job=build-linux)
+-   [MacOS
+    binary](https://gitlab.com/massalabs/massa/-/jobs/artifacts/testnet/download?job=build-darwin)
+
+## From source code
+Otherwise, if you wish to run a Massa node from source code, here are the steps to follow:
+
+### On Ubuntu / MacOS
+
+-   on Ubuntu, these libs must be installed:
+    `sudo apt install pkg-config curl git build-essential libssl-dev`
+-   install [rustup](https://www.rust-lang.org/tools/install):
+    `curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh`
+-   configure path: `source $HOME/.cargo/env`
+-   check rust version: `rustc --version`
+-   install
+    [nigthly](https://doc.rust-lang.org/edition-guide/rust-2018/rustup-for-managing-rust-versions.html):
+    `rustup toolchain install nightly`
+-   set it as default: `rustup default nightly`
+-   check rust version: `rustc --version`
+-   clone this repo:
+    `git clone --branch testnet https://gitlab.com/massalabs/massa.git`
+
+### On Windows
+
+#### Set up your Rust environment
+
+-   On Windows, you should first follow the indications from Microsoft
+    to be able to run on a Rust environment
+    [here](https://docs.microsoft.com/en-gb/windows/dev-environment/rust/setup).
+    -   Install Visual Studio (recommended) or the Microsoft C++ Build
+        Tools
+    -   Once Visual Studio is installed, click on C++ Build Tool. Select
+        on the right column called "installation details" the following
+        packages:
+        -   MSCV v142 -- VS 2019
+        -   Windows 10 SDK
+        -   C++ CMake tools for Windows
+        -   Testing Tools Core Feature
+    -   Click install on the bottom right to download & install those
+        packages
+-   Install Rust, to be downloaded
+    [here](https://www.rust-lang.org/tools/install)
+-   Install Git for windows, to be downloaded
+    [here](https://git-scm.com/download/win)
+
+#### Clone the Massa Git Repository
+
+-   Open Windows Power Shell
+    -   If the massa folder does not exist type:
+        `git clone https://gitlab.com/massalabs/massa.git`
+    -   Type: `cd massa`
+    -   Type: `git stash`
+    -   Type: `git checkout testnet`
+    -   Type: `git pull`
```

Comment:
Comment appart, these lines are very useless after a clone.

```md
- To clone the latest distributed version, type: `git clone --branch testnet https://github.com/massalabs/massa.git`
```

Reply:
This allows people to update an existing setup without having to back up their configs etc... in order to re-clone.
The "stash" part allows discarding any modifications to code-specific files.

Reply:
pull may fails if you changed something random in the config that conflicts with updates... it happened before :)

Reply:
No, the user config is in the gitignore, it's really strange that without a wrong usage the pull didn't worked. And furthermore, we stash. But we do we need to stash? Is there a stash pop? If not why not a reset?